components/email/src/Email.svelte

```diff
@@ -55,6 +60,26 @@
     ]) || {}) as EmailProperties;
   });
 
+  let contact;
+  $: (async () => {
+    if (thread) {
+      contact = await getContact(
+        thread.messages[thread.messages.length - 1].from[
+          thread.messages[thread.messages.length - 1].from.length - 1
+        ],
+      );
+    } else if (activeThread) {
+      contact = await getContact(
+        activeThread.messages[activeThread.messages.length - 1].from[
+          activeThread.messages[activeThread.messages.length - 1].from.length -
+            1
+        ],
+      );
+    } else if (message) {
+      contact = await getContact(message.from[message.from.length - 1]);
+    }
+  })();
```

Comment:
We'll not want to call this if no `component_id` is provided, since these requests will always fail.